packages/swapper/src/swappers/OneInchSwapper/endpoints.ts

```diff
@@ -53,6 +53,10 @@ export const oneInchApi: SwapperApi = {
 
     const { receiveAddress, affiliateBps } = tradeQuote
 
+    // TODO(gomes): when we actually split between TradeQuote and TradeRate in https://github.com/shapeshift/web/issues/7941,
+    // this won't be an issue anymore
+    if (!receiveAddress) throw new Error('receiveAddress is required for OneInchSwapper quotes')
```

Comment:
should we just delete this swapper since we aren't using anymore? also `isExecutableTradeQuote`

Reply:
Always in for some bloodbath. 8890a9f4ff4e7852f6e8703ecb1fed681ed691a2